links.py

```diff
@@ -0,0 +1,51 @@
+import requests

```

Comment:
Also, it is better to get a list of URL from the file as well as it was provided.

Reply:
Done